### PR TITLE
Update 09_persistent_configurations.md

### PR DESCRIPTION
**Pull request by mtspn**
- Update description of configmap changes

### CODE REVIEW

File: 101-lab/content/09_persistent_configurations.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ metadata:
 <kbd>![](./images/07_persistent_config_06.png)</kbd>
 
 #### Changing Config Map Content
-The content in your `configMap` can be changed and in most cases it's NOT dynamically updated in the pod. With that said, a new deployment will be required for the changes to be picked up.
+When a `configMap` is referenced by a deployment that uses the `rolling` deployment strategy, any changes to that `configMap` will be automatically applied to the pods created by that deployment. <!-- #TBC If your deployment uses the `recreate` strategy, the changes from your configmap will apply when you next roll out new pods.  -->
```
Comment (w8896699) [inline 101-lab/content/09_persistent_configurations.md:59]: I am not sure about the second part yet. But first part about rolling deployment is correct. And we should mention that this feature comes on 4.12 feature.

**Review by w8896699 — APPROVED** [review]

File: 101-lab/content/09_persistent_configurations.md (hunk 1 above)
Comment (mtspn) [reply 101-lab/content/09_persistent_configurations.md:59]: Okay, is there any documentation that I can point to that articulates this change? I couldn't find anything about this in the [4.12 release notes ](https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html)

Comment (w8896699) [reply 101-lab/content/09_persistent_configurations.md:59]: My bad, I counldn't find the evidence I read earlier, but here's the doc for this: https://kubernetes.io/docs/concepts/configuration/configmap/#mounted-configmaps-are-updated-automatically

Comment (mtspn) [inline 101-lab/content/09_persistent_configurations.md:59]:
```suggestion
When a mounted `configMap` is referenced by a deployment, any changes to that `configMap` will be [automatically applied to the pods created by that deployment](Trainer Cable Set for RadioMaster Radios
). These changes can take up to 2 minutes to be applied.
```

**Pull request merged**